views/index.blade.php

```diff
@@ -28,14 +28,37 @@
 
 <div id="graphiql">Loading...</div>
 <script src="{{ \MLL\GraphiQL\DownloadAssetsCommand::jsPath() }}"></script>
+<script src="{{ \MLL\GraphiQL\DownloadAssetsCommand::explorerPluginPath() }}"></script>
 <script>
+    var fetcher = GraphiQL.createFetcher({
+        url: '{{ url(config('graphiql.endpoint')) }}',
+        subscriptionUrl: '{{ config('graphiql.subscription-endpoint') }}',
+    });
+
+    function GraphiQLWithExplorer() {
+        var [query, setQuery] = React.useState(
+            '',
+        );
+
+        var explorerPlugin = GraphiQLPluginExplorer.useExplorerPlugin({
+            query: query,
+            onEdit: setQuery,
+        });
```

Comment:
Is this necessary or can we just simplify?

```suggestion
        var explorerPlugin = GraphiQLPluginExplorer.useExplorerPlugin();
```

Reply:
It seems that it needs them, so it can edit the query in the playground.

Reply:
Ah I see that they are used in two places - got it!